### DISCUSSION TITLE
Adding geopackage file to existing cloud project

### DISCUSSION THREAD

**Discussion by jacobthaqvist** [opengisch/QField#3563] [Q&A]
Hi!

Im having problems adding new files to a current cloud project. After adding the file in desktop and synchronizing with cloud the file appears under "files" in the cloud web interface. It does not appear in the mobile device though and synchronizing does not seem to help. 

Any ideas?

Thanks, 
Jacob

**Comment by UsuarioFabio** [comment]
Hi
I have same problem, any help please ?
Thanks
